features/customer_booking_request.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Customer creates a Booking Request
+  As a customer
+  I want to request an appointment
+  So that I can meet for guidance
```

Comment:
As these will be for phone appointments too, perhaps just `So I can find out more about my pension options` or something?